contracts/ERC900/ERC900BasicStakeContainer.sol

```diff
@@ -0,0 +1,132 @@
+pragma solidity ^0.4.24;
+
+import "./ERC900.sol";
+import "../ERC20/ERC20.sol";
+
+import "../library/SafeMath.sol";
+
+
+/**
+ * @title ERC900BasicStakeContainer
+ */
+contract ERC900BasicStakeContainer is ERC900 {
+  using SafeMath for uint256;
```

Comment:
Some gas optimization is possible here if we deploy the library separately and then point to it (as opposed to bundling it with both contracts)